CONTRIBUTING.md

```diff
@@ -13,3 +13,14 @@ This code of conduct applies both within project spaces and in public spaces whe
 Instances of abusive, harassing, or otherwise unacceptable behavior may be reported by opening an issue or contacting one or more of the project maintainers.
 
 This Code of Conduct is adapted from the Contributor Covenant, version 1.1.0, available at http://contributor-covenant.org/version/1/1/0/
+
+### Contributing
+
+1. Fork the repository
+2. Create your feature branch (`git checkout -b my-new-feature`)
+3. Install dependencies (`composer install`)
+4. Cigar uses [Kahlan](https://kahlan.github.io/docs/) as test runner. So, please make sure it's already installed
```

Comment:
Thanks for this! Small note: Kahlan should be installed with the other dev composer dependencies when running `composer install` - so this step isn't necessary (but maybe put a link in the step on running kahlan)